allms/models/vertexai_base.py

```diff
@@ -47,3 +48,56 @@ def was_response_blocked(generation: Generation) -> bool:
             llm_output=result.llm_output,
             run=result.run
         )
+
+
+class VertexAIModelGardenWrapper(VertexAIModelGarden):
+    temperature: float = 0.0
+    max_tokens: int = 128
+    top_p: float = 0.95
+    top_k: int = 40
+    n: int = 1
+    allowed_model_args: Optional[List[str]] = ["temperature", "max_tokens", "top_p", "top_k", "n"]
+
+    @property
+    def _default_params(self) -> Dict[str, Any]:
+        return {
+            "temperature": self.temperature,
```

Comment:
these params and `allowed_model_args` are the same. Can we think about anyway to not have code repetitions? Probably also other models defined in `models` may take advantage of defining these vars more "globally".

Reply:
Done